### DISCUSSION TITLE
[V5] Luxon error when using version 1.26.0

### DISCUSSION THREAD

**Discussion by luisnovaisdev** [adonisjs/core#2513] [Help] [answered]
Give me this error when using the last luxon version when use DateTime in model 

E_INVALID_DATE_COLUMN_VALUE: The value must be an instance of "luxon.DateTime"

**Comment by thetutlage** [comment, marked as answer]
Fixed here https://github.com/adonisjs/lucid/commit/f283738a60ed0441af0e3ec3f4800572c42efe35. Not released yet. For now, downgrading the luxon version to one that Lucid uses is the only option.

Post Lucid release, you can freely have different versions of Luxon

**Reply by arthurcvm** [reply]
Thanks @thetutlage! The version 1.25.0 solve my problem.
